src/Plugin/Condition/EntityHasIdentifier.php

```diff
@@ -108,8 +111,7 @@ public function evaluate(): bool {
     }
 
     $entity = $this->getContextValue('entity');
-    if ($entity instanceof FieldableEntityInterface) {
-      $identifier = $this->entityTypeManager->getStorage('dgiactions_identifier')->load($this->configuration['identifier']);
+    if ($entity instanceof FieldableEntityInterface && ($identifier = $this->getIdentifier())) {
```

Comment:
Strictly, `$identifier = $this->getIdentifier()` _should_ never be false-y, but let's allow it anyhow.

In a sense, this is much the same as what #49 did.